drift_dev/lib/src/analysis/results/view.dart

```diff
@@ -96,6 +96,6 @@ class DartViewSource extends DriftViewSource {
   final AnnotatedDartCode dartQuerySource;
   final TableReferenceInDartView? primaryFrom;
   final List<TableReferenceInDartView> staticReferences;
-
-  DartViewSource(this.dartQuerySource, this.primaryFrom, this.staticReferences);
+  final String? staticSource;
```

Comment:
There should be a comment explaining that this is used when referring to an element not yet generated.

Reply:
Done